website/docs/diff.md

```diff
@@ -23,12 +23,11 @@ slug: /diff/
    * update operations producing `Infinity`, `-Infinity`, or `NaN` are not supported.
 8. Database and collection names restrictions:
    * name cannot start with the reserved prefix `_ferretdb_`;
-   * database name must not include non-latin letters, spaces, dots, dollars;
-   * collection name must not include non-latin letters, spaces, dots or dollars;
-   * name must not start with a number;
+   * database name must not include non-latin letters, spaces, dots;
```

Comment:
dollar is not allowed in both ferretdb and mongodb so it's not diff. It's already tested https://github.com/FerretDB/FerretDB/blob/main/integration/basic_test.go#L418